jackson/src/test/java/io/gravitee/definition/jackson/api/DebugApiDeserializerTest.java

```diff
@@ -58,7 +58,7 @@ public void debugApi_withResponse() throws Exception {
     public void debugApi_withDebugSteps() throws Exception {
         DebugApi debugApi = load("/io/gravitee/definition/jackson/debug/debug-api-with-debug-steps.json", DebugApi.class);
 
-        assertEquals(debugApi.getDebugSteps().size(), 10);
+        assertEquals(debugApi.getDebugSteps().size(), 11);
```

Comment:
I would have prefered to create a new test case (dedicated file and method)

Reply:
with dedicated resource ? like `debug-api-with-debug-steps-errors.json`